airbyte-integrations/connectors/source-hubspot/source_hubspot/client.py

```diff
@@ -37,9 +37,9 @@ def __init__(self, start_date, credentials, **kwargs):
             "campaigns": CampaignStream(**common_params),
             "companies": CRMObjectIncrementalStream(entity="company", associations=["contacts"], **common_params),
             "contact_lists": ContactListStream(**common_params),
-            "contacts": CRMObjectIncrementalStream(entity="contact", **common_params),
+            "contacts": CRMObjectIncrementalStream(entity="contact", associations=["companies"], **common_params),
             "deal_pipelines": DealPipelineStream(**common_params),
-            "deals": DealStream(associations=["contacts"], **common_params),
+            "deals": DealStream(associations=["contacts", "companies"], **common_params),
```

Comment:
Adding new associations change the response from the API. You need to change the schema file to be compatible with the new response.

Reply:
Ah of course @marcosmarxm! 

I've updated the schemas to include the relevant associations. Seems like the `deals` stream schema has always had the `companies` associations so I didn't add that.